cmd/start.go

```diff
@@ -737,13 +698,80 @@ func (x *Start) Execute(args []string) error {
 	return nil
 }
 
+func getConfigFiles(datafolder string) error {
+	var err error
+	mainConfig, err = ioutil.ReadFile(path.Join(datafolder, "config"))
+	if err != nil {
+		log.Error("read config:", err)
+		return err
+	}
+
+	apiConfig, err = schema.GetAPIConfig(mainConfig)
+	if err != nil {
+		log.Error("scan api config:", err)
+		return err
+	}
+	torConfig, err = schema.GetTorConfig(mainConfig)
+	if err != nil {
+		log.Error("scan tor config:", err)
+		return err
+	}
+	dataSharingConfig, err = schema.GetDataSharing(mainConfig)
+	if err != nil {
+		log.Error("scan data sharing config:", err)
+		return err
+	}
+	dropboxToken, err = schema.GetDropboxApiToken(mainConfig)
+	if err != nil {
+		log.Error("scan dropbox api token:", err)
+		return err
+	}
+	resolverConfig, err = schema.GetResolverConfig(mainConfig)
+	if err != nil {
+		log.Error("scan resolver config:", err)
+		return err
+	}
+	republishInterval, err = schema.GetRepublishInterval(mainConfig)
+	if err != nil {
+		log.Error("scan republish interval config:", err)
+		return err
+	}
+	walletsConfig, err = schema.GetWalletsConfig(mainConfig)
+	if err != nil {
+		log.Error("scan wallets config:", err)
+		return err
+	}
+	return nil
+}
```

Comment:
This is a nice step. Next would be to formalize the config struct, remove the globals, and make `getConfigFiles` return an instantiated config. This removes mutation and global vars from the config handling as well as makes unit testing this possible.